snmp/datadog_checks/snmp/data/default_profiles/cisco-nexus.yaml

```diff
@@ -30,6 +30,22 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.1
           name: entSensorType
         tag: sensor_type
+        mapping:
+          1: other
+          2: unknown
+          3: volts_ac
+          4: volts_dc
+          5: amperes
+          6: watts
+          7: hertz
+          8: celsius
+          9: percent_rh
+          10: rpm
+          11: cmm
+          12: truthvalue
+          13: special_enum
+          14: d_bm
+          15: d_b
```

Comment:
We can keep these as `db` and `dbm` since they are units and it makes more sense to just lowercase the value